instrument.sh

```diff
@@ -141,4 +141,5 @@ if [ "$ENABLE_PROFILING" = "true" ]; then
 
   # Configure the bytecode instrumentation configuration file
   export OTEL_DOTNET_AUTO_INTEGRATIONS_FILE="$OTEL_DOTNET_AUTO_HOME/integrations.json"
+  export OTEL_DOTNET_AUTO_PLUGINS="Splunk.OpenTelemetry.AutoInstrumentation.Plugin, Splunk.OpenTelemetry.AutoInstrumentation, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null"
```

Comment:
@Kielek I see that MinVer works only for NuGet packages. Shouldn't we use

```
  <PropertyGroup>
    <Version>PUT.VERSION.HERE</Version>
  </PropertyGroup>
```

instead?

Reply:
@pjanotti @RassK any thoughts?

Reply:
Easier only in a sense that we would not have to change version in 2 places with every release, imo we should do that anyway.

Reply:
MinVer is working as expected also for libraries.
FileVersion != Assembly version. See https://github.com/adamralph/minver#version-numbers and https://learn.microsoft.com/en-ca/dotnet/standard/library-guidance/versioning#version-numbers

Reply:
I think that @RassK said somwhere that this line could be simplified to:


```suggestion
  export OTEL_DOTNET_AUTO_PLUGINS="Splunk.OpenTelemetry.AutoInstrumentation.Plugin, Splunk.OpenTelemetry.AutoInstrumentation"
```